centos-bootc-config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "Labels": {
+        "org.opencontainers.image.redhat.rhel.compose-id": "CentOS-Stream-9-20240131.d.0",
+        "org.opencontainers.image.id": "CentOS-Stream",
+        "org.opencontainers.image.version-id": "9"
```

Comment:
I don't think we should prefix things with `org.opencontainers` that aren't official standards as a general rule.

There is outstanding discussion about this, xref https://github.com/CentOS/centos-bootc/issues/110 :arrow_right: https://github.com/opencontainers/image-spec/issues/1152

`org.opencontainers.image.redhat.rhel.compose-id` as a label is especially weird because it's clearly Fedora-derivative specific.  Why not just `redhat.compose-id`?

Reply:
Makes sense. Will update the PR.